### PR TITLE
Fix bot to be usable

### PR DESCRIPTION
**Pull request by puppy0cam**
Core features of the bot were non-functional. This PR resolves these issues to make the bot functional again.

### CODE REVIEW

File: src/handler/messagehandler.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default class MessageHandler {
         img = await new Promise((resolve) => {
           // @ts-ignore
           request.get(attachment.url, async (err: any, res: any, body: Buffer) => {
-            if (body !== undefined) {
+            if (body != null) {
```
Comment (AltF02) [inline src/handler/messagehandler.ts:61]: Why `!=` and not `!==`?

File: src/commands/events/linkevent.ts [hunk 2]
```diff
@@ -33,15 +33,15 @@ export = class CreateEvent extends Command {
     });
   }
 
-  async run(msg: CommandoMessage, { name, channel, reviewChannel }: {name: string, channel: string, reviewChannel: string}) {
+  async run(msg: CommandoMessage, { name, channel, review_channel }: {name: string, channel: string, review_channel: string}) {
```
Comment (AltF02) [inline src/commands/events/linkevent.ts]: Please use camelcase

**Review by AltF02 — CHANGES_REQUESTED** [review]
Could you possibly not be so aggressive and disrespectful?

File: src/handler/messagehandler.ts (hunk 1 above)
Comment (puppy0cam) [reply src/handler/messagehandler.ts:61]: I believe in runtime I was finding it to be null, while the code was checking for undefined. I went for the best of both worlds and went with an assertation that accounts for both null as well as undefined.

**Comment by puppy0cam** [comment]
> Could you possibly not be so aggressive and disrespectful?

My apologies

**Comment by AltF02** [comment]
Cheers

**Pull request merged**